Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -227,6 +227,27 @@ theorem isPrincipal_inv (I : FractionalIdeal R₁⁰ K) [Submodule.IsPrincipal (
   exact (right_inverse_eq _ I (spanSingleton _ (generator (I : Submodule R₁ K))⁻¹) hI).symm
 #align fractional_ideal.is_principal_inv FractionalIdeal.isPrincipal_inv
 
+variable {K}
+
+lemma den_mem_inv {I : FractionalIdeal R₁⁰ K} (hI : I ≠ ⊥) :
+    (algebraMap R₁ K) (I.den : R₁) ∈ I⁻¹ := by
+  rw [mem_inv_iff hI]
+  intro i hi
+  rw [← Algebra.smul_def (I.den : R₁) i, ← mem_coe, coe_one]
+  suffices Submodule.map (Algebra.linearMap R₁ K) I.num ≤ 1 from
+    this <| (den_mul_self_eq_num I).symm ▸ smul_mem_pointwise_smul i I.den I.coeToSubmodule hi
+  convert map_mono (show I.num ≤ 1 by simp only [Ideal.one_eq_top, le_top, bot_eq_zero])
```

Comment:
```suggestion
  apply le_trans <| map_mono (show I.num ≤ 1 by simp only [Ideal.one_eq_top, le_top, bot_eq_zero])
```
I'm always a bit nervous of `convert` followed by an argument with a little content, because `convert` is so dependent on definitional equality. My suggestion makes this proof a little more robust.